models/aws_cloud_cost__overview.sql

```diff
@@ -231,7 +231,7 @@ final as (
             'product_to_region_code',
             'product_instance_family',
             'product_instance_type'
-        ]) }} as unique_key
+        ]) }} as unique_key 
```

Comment:
So it seems this model has a unique key that includes every single dimension. Do we need to consider any passthrough columns in the unique key as well?

Reply:
ah good catch